viewer/tsconfig.json

```diff
@@ -12,6 +12,8 @@
     "noUnusedParameters": true,
     // Strict Type-Checking Options
     "strict": true,
+	"strictNullChecks": false,
+	"noImplicitAny": false,

```

Comment:
Why? We should avoid this if we can

Reply:
`geo-three` library/package causes error when viewer is build without these option. This option not required by our tool/examples but by the library

Reply:
Hmm, should be possible to exclude modules from checks - can you checkout https://www.typescriptlang.org/tsconfig#exclude?

Reply:
> * `GeomapTool.setMapProvider` is provided for examples to show different map options

I think we just can create a new tool instance for this case and dispose the old one (?)